cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -1222,6 +1371,20 @@ Replaced by:
 ----
 ON HOME GRAPH
 ----
+
+a|
+label:syntax[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+MATCH (a) RETURN (a)--()
+----
+a|
+Pattern expressions producing lists of paths are deprecated. Use a pattern comprehension instead. Pattern expressions can still be used as existence predicates in `WHERE` clauses and in `exists()`. Replaced by:
```

Comment:
Same suggestion from above:

```suggestion
Pattern expressions producing lists of paths are no longer supported, but they can still be used as existence predicates in `WHERE` clauses.
Instead, use pattern comprehension:
```

Reply:
Similar to the one above, this is in the table for 4.3 so it should say deprecated but I will try to reformulate it to something more like this. As for the `exists()` part, the `exists()` function has been removed in 5.0, but it is still valid when talking about 4.3, so I think I prefer to keep it here